packages/atomic/src/pages/index.html

```diff
@@ -215,6 +215,7 @@
             <atomic-notifications></atomic-notifications>
           </atomic-layout-section>
           <atomic-layout-section section="results">
+            <atomic-generated-answer></atomic-generated-answer>
             <atomic-smart-snippet></atomic-smart-snippet>
```

Comment:
do we ever get both a smart snippet and a generated answer at the same time?

Reply:
Good question that kinda goes beyond the scope of the PR. I don't think we have a definitive answer as to how the two features coexiste at the moment. But in any case I didn't mean to leave my component in here. I removed it.